README.md

```diff
@@ -291,32 +289,32 @@ This context manager allows you to intercept any events emitted by the framework
 Usage:
 
 ```python
-from ops.charm import StartEvent, UpdateStatusEvent
-from scenario import State, Context, DeferredEvent, capture_events
+import ops
+import scenario
 
-with capture_events() as emitted:
-    ctx = Context(...)
+with scenario.capture_events() as emitted:
```

Comment:
Similarly, I'm uncertain whether/why we need this API. Probably a Pietro discussion.